e2e/cypress/support/index.js

```diff
@@ -15,6 +15,7 @@
 
 // Import commands.js using ES2015 syntax:
 import './commands'
+import 'cypress-pipe'
```

Comment:
Why is this added? What does it do? Is it necessary?

Reply:
import 'cypress-pipe' removed.